pages/windows/explorer.md

```diff
@@ -0,0 +1,15 @@
+# explorer
+
+> The Windows file explorer.
```

Comment:
The rest of the page uses capital E for Explorer. I think we should use File Explorer here too.